modules/fundamental/src/sbom/service/sbom.rs

```diff
@@ -139,6 +139,9 @@ impl SbomService {
                     .alias("sbom_node", "r0")
                     .translator(|f, op, v| match f.split_once(':') {
                         Some(("label", key)) => Some(format!("labels:{key}{op}{v}")),
+                        Some(("version", _key)) => {
+                            Some(format!("versions @> ARRAY[{}]::text[]", v))
+                        }
```

Comment:
Whatever you think this is doing, it's not. :stuck_out_tongue_winking_eye: 

If you want to alias the field "version" to the actual field "versions", you could do...

```suggestion
                        None => match f {
                            "version" => Some(format!("versions{op}{v}")),
                            _ => None,
                        }
```

Reply:
@jcrossley3, 
Oh my, and I thought I was doing something useful with that ! 
I still have a split brain between the versions search and the full text search.
Thanks for the suggestion, just added it.